https/gifts.js

```diff
@@ -5,113 +5,56 @@ also has functions to return admin stuff as well
 */
 const functions = require('firebase-functions');
 require('firebase/firestore');
-const { getUUID, getGeneralQueries } = require('../utils/utils');
-const { YEAR } = require("../config/constants");
+const { getUUID, getGeneralQueries, markGifteeAccount } = require('../utils/utils');
+const { EVENT } = require("../config/constants");
 const db = require('../config/db');
 
 // marks gift as went
-const sendGift = functions.https.onCall(async({user}, context) => {
-  // marks their own account
+const sendGift = functions.https.onCall(async({user, value, giftee_uuid}, context) => {
```

Comment:
What is "value"?

Reply:
Probally clould have nbamed it better, this is a bool that sets the sent/recieved/reported flag, previous version only set it to true which meant that you couldnt undo it

Reply:
Makes sense, good upgrade!